server/models/orderModel.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+
+const orderSchema = mongoose.Schema(
+  {
+    user: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    orderItems: [
+      {
+        name: { type: String, required: true },
+        quantity: { type: Number, required: true },
+        image: { type: String, required: true },
+        price: { type: Number, required: true },
+        id: {
+          type: mongoose.Schema.Types.ObjectId,
+          required: true,
+          ref: "Recipe",
+        },
+      },
+    ],
+    chefId: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,

```

Comment:
@Vaibhav-Kambli you need to have a ref for the chefId like all other object type field